msteams-platform/feedback.md

```diff
@@ -2,28 +2,38 @@
-|          **General questions**         |          General questions about the Microsoft Teams developer platform, please send an email to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned above and use email only if none of the above are applicable.                                                                                                                                                                          |
+|         **Community support**          |<ol><li> Questions can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.  Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. New users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask) for more details. </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
+|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.                                                                                                                                                                                                                       |
+|            **Bot & Messaging Extension SDK issues/suggestions**             |      Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |
```

Comment:
Use the `botframework & microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams).